CMakeLists.txt

```diff
@@ -19,9 +19,17 @@ set(CMAKE_CXX_STANDARD 11)
 if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Release)
 endif()
-set(CMAKE_CXX_FLAGS_DEBUG "-g -march=native")
```

Comment:
Could you keep the ˋ-march=nativeˋ flag for the debug build? When dependencies are installed in release with this flag, this would cause trouble.